shapeout/gui/plot_scatter.py

```diff
@@ -143,9 +143,6 @@ def scatter_plot(measurement,
         plot_kwargs["type"] = "cmap_scatter"
         plot_kwargs["color_mapper"] = ca.jet
 
-    if pd.get_data("index") is not None:
```

Comment:
Something went terribly wrong here. Your pull request introduces a regression (#86)!
You have two options:
1. Change these lines back the way they were in your pull request.
2. Close this pull request, delete your ShapeOut fork, create a new one, add your changes and submit a new pull request.